config/stylelint/.stylelintrc

```diff
@@ -10,7 +11,6 @@
     "at-rule-no-unknown": null,
     "block-no-empty": true,
     "no-descending-specificity": null,
-    "function-calc-no-invalid": true,
```

Comment:
Das brauchen wir - benutzen wir die calc() in less, meckert CS ansonsten.

Reply:
ja aber https://stylelint.io/migration-guide/to-14/#rules-deprecated-in-1370